common/djangoapps/third_party_auth/api/tests/test_views.py

```diff
@@ -130,3 +135,83 @@ def test_list_connected_providers__withapi_key(self, api_key, target_user, expec
         if expect_result == 200:
             self.assertIn("active", response.data)
             self.assertItemsEqual(response.data["active"], self.expected_active(target_user))
+
+    @ddt.data(
```

Comment:
Please create a new Test class for testing the `UserMappingView`.  You can rename the existing one from `ThirdPartyAuthAPITests` to something like `TestUserView`.  You can also move any common `setUp`s and helpers into a common Test base or mixin class.